ui/pages/multichain-accounts/wallet-details/wallet-details.component.test.tsx

```diff
@@ -265,7 +265,7 @@ describe('WalletDetails', () => {
       );
     });
     // does not show backup reminder when seed phrase is backed up
-    expect(queryByText('Backup')).not.toBeInTheDocument();
+    expect(queryByText('Back up')).not.toBeInTheDocument();
```

Comment:
Had a merge conflict for the `backup` message, designs point to it to being with a space "Back up" instead of "Backup". I adjusted this test to account for that as a result.